### DISCUSSION TITLE
Flags for 'jira me' to get displayName and emailAddress

### DISCUSSION THREAD

**Discussion by limarkxx** [ankitpokhrel/jira-cli#684] [Ideas]
I sometimes use `http GET https://gubagoo.atlassian.net/rest/api/2/myself | jq '.displayName,.emailAddress' -r` to set my Git user.name and user.email config.

**Comment by ankitpokhrel** [comment]
Hi @limarkxx, `jira me` command currently returns the cached login name field from the config and is not making a call to  `/myself` endpoint. Will wait to see if there is any other usecases before adding new flags. Anyway, PRs are welcome if you wish to work on it.
